parser/parser.y

```diff
@@ -232,6 +235,7 @@ import (
 	addTime				"ADDTIME"
 	admin				"ADMIN"
 	aesDecrypt			"AES_DECRYPT"
+	always				"ALWAYS"
```

Comment:
You need add them to 
`UnReservedKeyword:`
and 
`ReservedKeyword:` 
in parser.y

And also add them to `TestSimple` in parser_test.go

If you're not sure whether a ident is keyword or not, look up it here:
https://dev.mysql.com/doc/refman/5.7/en/keywords.html